content/docs/gke/gcp-e2e.md

```diff
@@ -463,13 +415,13 @@ a [Docker][docker] container image and push the image to
     export VERSION_TAG=$(date +%s)
     ```
 
-1. Set the path in Container Registry that you want to push the image to:
+2. Set the path in Container Registry that you want to push the image to:
```

Comment:
Please revert this change, and all the similar changes from `1.` to a specific number below. We keep the list formatting as just `1.` to avoid having to renumber when we add/remove steps.